Documentation/githooks.txt

```diff
@@ -496,23 +496,25 @@ This hook is invoked by `git-p4 submit`. It takes no parameters and nothing
 from standard input. Exiting with non-zero status from this script prevent
 `git-p4 submit` from launching. Run `git-p4 submit --help` for details.
 
-post-index-change
```

Comment:
Perhaps this is an incorrect merge-conflict resolution? I'm guessing we don't want to delete the section about the post-index-change hook.

Reply:
Good call!

Reply:
(And yes, this was a merge conflict resolution, I remember it... I had thought that there was already the correct section and I had removed the duplicate, incorrect section...)